release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1335,7 +1368,9 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *OLM*
 
+* Previously, `CustomResourceDefinition` (CRD) objects applied as part of an Operator installation could sometimes satisfy the installation requirements of a newer version of the same Operator. Consequently, during an Operator upgrade, the version being replace could be prematurely removed. In some cases, the upgrade would stop. With this update, the CRDs that are created or updated as part of the Operator bundle installation are annotated to indicate their bundle of origin. These annotations are used by the `ClusterServiceVersion` (CSV) controller to distinguish between pre-existing CRDs and same-bundle CRDs. As a result, upgrades will not complete until the current version's CRDs have been applied. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1947946[*BZ#1947946*])
 
+* Previously, `Catalog Registry` pods did not have `readOnlyRootFileSystem: false` explicitly set in their `securityContext` field. Consequently, if an SCC existed that enforced `readOnlyRootFileSystem: true` and matched the `securityContext` of the `Catalog Registry` pod, it would be assigned to the `Catalog Registry` pod and cause it to fail repeatedly. This update explicitly sets `readOnlyRootFileSystem: false` in the `securityContext` field. As a result, `Catalog Registry` pods are no longer matched to SCCs that enforce a read-only root filesystem and no longer fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1961472[*BZ#1961472*])
```

Comment:
s/`Catalog Registry` pods/Catalog Registry pods. Are the pods named _Catalog Registry_? I don't see `Catalog Registry` pods in the docs and don't see a `Catalog Registry` pod in one cluster I looked at.